dominion/settings.py

```diff
@@ -218,25 +218,39 @@
 }
 # -------------------------------------------------------------------------------------
 # Showcase Stories
-
 HURUMAP['showcase_stories'] = [
+    {
+        'index': 0,
+        'title':  'Deforestation: 48 Years of Kenya\'s Unspoken Disaster',
+        'author': 'Africa Uncensored',
+        'brief':  'Deforestation in Kenya, how did it happen? Who is responsible?',
+        'link':   '',
+        'date': '05 Apr 2019',
+        'mediaSrc':   'https://www.youtube.com/embed/PDhuK4KrIoU',
+        'media': 'iframe',
+        'country': 'kenya'
+    },
```

Comment:
These should move to a gSheet... On a different story/PR so we don't hold this PR back of course.

Reply:
@kilemensi alright. I will create a story on PT